### PR TITLE
required changes to efiling-demo

### PR DESCRIPTION
**Pull request by gpoleze**

### CODE REVIEW

**Review by BrendanBeachBC — APPROVED** [review]
Looks great!

**Review by TayGov — APPROVED** [review]
👍

File: src/frontend/efiling-frontend/src/components/page/upload/__snapshots__/Upload.test.js.snap [hunk 1]
```diff
@@ -1778,7 +1778,7 @@ exports[`Upload Component successful document upload works as expected 1`] = `
                           <select
                             class="bcgov-dropdown"
                             data-testid="dropdown"
-                            id="file-dropdown"
+                            id="dropdown"
                             name="options"
                           >
```
Comment (TayGov) [inline src/frontend/efiling-frontend/src/components/page/upload/__snapshots__/Upload.test.js.snap:1783]:
```suggestion
                          >
                          
```

File: src/frontend/efiling-frontend/src/components/page/upload/__snapshots__/Upload.test.js.snap [hunk 2]
```diff
@@ -1778,9 +1778,10 @@ exports[`Upload Component successful document upload works as expected 1`] = `
...
+
```
Comment (TayGov) [inline src/frontend/efiling-frontend/src/components/page/upload/__snapshots__/Upload.test.js.snap:1784]:
```suggestion
```

**Review by TayGov — APPROVED** [review]
Retriggered checks that wouldn't complete

**Pull request closed without merging**